doc/source/notebooks/advanced/varying_noise.pct.py

```diff
@@ -114,10 +115,15 @@ def variational_expectations(self, Fmu, Fvar, Y):
 def objective_closure():
     return - model.log_marginal_likelihood()
 
-opt = gpflow.optimizers.Scipy()
-opt.minimize(objective_closure,
-             model.trainable_variables,
-             options=dict(maxiter=ci_niter(1000)))
+natgrad_opt = NaturalGradient(gamma=1.0)
+scipy_opt = gpflow.optimizers.Scipy()
+
+model.q_mu.set_trainable(False)
```

Comment:
this is not allowed, you should do the following
```
from gpflow.utilities import set_trainable
set_trainable(model.q_mu, False)
```

Probably best to run the notebook and visually check the ipynb before committing to catch these errors.

Reply:
oh yeah

do we not have tests that check the notebook doesn't error?

Reply:
yes we do, and they failed :)